python/katana/bug/environment.py

```diff
@@ -242,6 +243,15 @@ def capture_environment(filename: Optional[Union[str, Path, Any]] = None):
 
             if get_ipython():
                 # If IPython is importable and actually loaded, display the link.
+                try:
+                    # Path.readlink doesn't exist before Python 3.9, so that cannot be used quite yet.
+                    ipython_root_directory = os.readlink(f"/proc/{os.environ['JPY_PARENT_PID']}/cwd")
+                except OSError:
+                    warnings.warn(
+                        "Could not determine Jupyter root path. Guessing that it is the current working "
+                        "directory. The download link may be incorrect."
```

Comment:
how about providing some manual way to determine the correct download link here? Like, "copy path of notebook" and use the following templated link:.... or the output of `!pwd` within the notebook

Reply:
That's a good idea. Hmm. Though I actually think this will only happen if the code is running in ipython, but not Jupyter. So I wouldn't know what instructions to give.

Reply:
I think I'm gonna leave that for another time. I don't know what I would actually add.